pkg/connector/connector.go

```diff
@@ -57,9 +62,10 @@ func (c *Calendly) Validate(ctx context.Context) (annotations.Annotations, error
 }
 
 // New returns a new instance of the connector.
-func New(ctx context.Context, token string) (*Calendly, error) {
+func New(ctx context.Context, cfg *viper.Viper) (*Calendly, error) {
```

Comment:
Revert `New()` to the previous method signature so that it's easier to update in C1.

Reply:
Fixed